example/radar/radar_sweep_hdf5.py

```diff
@@ -0,0 +1,94 @@
+"""
+=====
+About
+=====
+Example for DWD radar sites OPERA HDF5 (ODIM_H5) format using wetterdienst and wradlib.
+
+See also:
+- https://docs.wradlib.org/en/stable/notebooks/fileio/wradlib_radar_formats.html#OPERA-HDF5-(ODIM_H5)
+
+This program will request the latest RADAR SWEEP_VOL_PRECIPITATION_V data
+for Boostedt and plot the outcome with matplotlib.
+
+
+=====
+Setup
+=====
+::
+
+    brew install gdal
+    pip install wradlib
+
+"""
+import logging
+from tempfile import NamedTemporaryFile
+
+import numpy as np
+import wradlib as wrl
+import matplotlib.pyplot as pl
+
+from wetterdienst.dwd.radar.metadata import RadarParameter, RadarDate, RadarDataType
+from wetterdienst.dwd.radar.sites import RadarSite
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger()
+
+from wetterdienst import DWDRadarRequest
+
+
+def plot(data: np.ndarray):
+    """
+    Convenience function for plotting radar data.
+    """
+
+    fig = pl.figure(figsize=(10, 8))
+    im = wrl.vis.plot_ppi(data['dataset1/data1/data'], fig=fig, proj='cg')
+
+
+def radar_info(data: dict):
+    """
+    Display data from radar request.
+    """
+    print("Keys:", data.keys())
+
+    log.info("Data")
+    for key, value in data.items():
+        print(f"- {key}: {value}")
+
+
+def radar_hdf5_example():
+
+    log.info("Acquiring radar sweep data in HDF5")
+    request = DWDRadarRequest(
+        radar_parameter=RadarParameter.SWEEP_VOL_PRECIPITATION_V,
+        date_times=RadarDate.LATEST.value,
+        radar_site=RadarSite.BOO,
+        radar_data_type=RadarDataType.HDF5,
+    )
```

Comment:
This will not discriminate between **simple** and **polarimetric** yet.

Reply:
I think we can solve this on two ways: 
- One would be to introduce another flag/enum 
- Or (my favorite) to introduce new radar_parameters. 
SWEEP_VOL_PRECIPITATION_V_SIMPLE and SWEEP_VOL_PRECIPITATION_V_POLAR
And adding a note in the documentation.

Reply:
@meteoDaniel I'm not sure I get this. Would it not be:

- SWEEP_VOL_V_SIMPLE and SWEEP_VOL_V_POLAR
- SWEEP_PRECIPITATION_V_SIMPLE and SWEEP_PRECIPITATION_V_POLAR

Reply:
Thanks. Let's do variant b) and extrapolate these options into the list of RadarParameters, as both of you suggested.

Reply:
I've just looked at this in more depth. The problem with the distinguation on the `RadarParameter` level would be that this difference only appears for the data in HDF5-format, right? The data in the main folder (e.g. [1]) does not distinguish between both variants.

So, while I definitively would have preferred the variant to encode this into `RadarParameter`, we might be better off to use just **another** parameter for this, which is only applied when selecting the `RadarDataType.HDF5` variant.

What do you think?

[1] https://opendata.dwd.de/weather/radar/sites/sweep_pcp_v/boo/

Reply:
With 99749ab, only these tests will fail:
```
tests/dwd/radar/test_api_historic.py::test_radar_request_site_historic_sweep_precipitation_hdf5_yesterday FAILED
tests/dwd/radar/test_api_historic.py::test_radar_request_site_historic_sweep_precipitation_hdf5_timerange FAILED
```

The reason for that is that **simple** and **polarimetric** is not taken into account yet. I will introduce a respective parameter within the next iteration.

Reply:
I've added a new argument `subset` to the machinery with 27a0b49.